src/main/resources/steps/composite.steps

```diff
@@ -15,3 +15,33 @@ Then the text 'Log out' exists
 When I click on element located by `By.xpath(//button[@data-testid="logout-button"]/span)`
 When I wait until the page title contains the text 'Manage Your Team'
 Then the page with the URL 'https://trello.com/home' is loaded
+
+Composite: When I delete board with `$boardname`
+Then number of elements found by `By.xpath(//h2[text()='Your boards'])` is EQUAL_TO `1`
+When I hover mouse over element located `By.xpath(//ul[@data-testid="collapsible-list-items"]//a[@title="<boardname> (currently active)"])`
```

Comment:
pls avoid using the elements names in locators until there is a need of doing it.
Here instead of //ul[@data-testid="collapsible-list-items"]//a[@title="<boardname> (currently active)"]
better to use
//*[@data-testid="collapsible-list-items"]//*[@title="<boardname> (currently active)"]

PLease apply for all locators in the composite.steps file